### PR TITLE
Remove www. from some

### PR DESCRIPTION
**Pull request by Rexadev**

### CODE REVIEW

File: porn [hunk 1]
```diff
@@ -173,45 +173,45 @@
...
-0.0.0.0 www.dasdas.jp
-0.0.0.0 www.digital-ark.com
-0.0.0.0 www.dmm.co.jp
-0.0.0.0 www.dogma.co.jp
-0.0.0.0 www.dse-web.com
-0.0.0.0 www.dt01.co.jp
-0.0.0.0 www.e-corona.com
-0.0.0.0 www.elog-ch.net
-0.0.0.0 www.emmanuelle-av.com
+0.0.0.0 av-a-box.com
```
Comment (spirillen) [inline porn:176]:
```
 whois av-a-box.com
No match for domain "AV-A-BOX.COM".
>>> Last update of whois database: 2021-11-29T07:48:20Z <<<
```

how did you check this record?

Comment (spirillen) [reply porn:176]: image

File: porn [hunk 2]
```diff
@@ -308,60 +308,60 @@
...
+0.0.0.0 wanz-factory.com
+0.0.0.0 woman-otona.com
+0.0.0.0 yamiboard.com
+0.0.0.0 yamigama.com
+0.0.0.0 yellow-movie.com
+0.0.0.0 yellowbox.co.jp
+0.0.0.0 yellowduck.jp
+0.0.0.0 zetton-av.com
+0.0.0.0 zeus-web.net
+0.0.0.0 giga-jp.com
```
Comment (spirillen) [inline porn:364]: You do realize this domain solely serves it contents over `www.` right?

other wise check the tested record here: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1788

File: porn [hunk 3]
```diff
@@ -308,60 +308,60 @@
...
+0.0.0.0 rocket-inc.net
+0.0.0.0 rookie-av.jp
+0.0.0.0 royalart.co.jp
+0.0.0.0 ruby-av.com
+0.0.0.0 s-cute.com
+0.0.0.0 s1s1s1.com
+0.0.0.0 sadistic-v.com
+0.0.0.0 saiminou.com
+0.0.0.0 sakurakikaku.net
+0.0.0.0 sexia.tv
```
Comment (spirillen) [inline porn:320]: sexia.tv                                                                                             INACTIVE    STDLOOKUP  Unknown           Unknown    AVAILABILITY

image

**Review by spirillen — CHANGES_REQUESTED** [review]
Just those I've marked from random picks, are turn up as dead domains. so how have you been able to determine they should be blacklisting (blocking) anything just by flipping the the usage of `www.`???  I'm really interested as your suggestion are the opuses of any test I can provide.

Just think of:
- https://github.com/tiuxo/hosts/pull/33
- https://github.com/tiuxo/hosts/pull/30
- https://github.com/tiuxo/hosts/pull/34

**If** you are in any doubt on how hosts and pihole files have to be constructed, please just do ask.

**If** you actually can provide information (proof) that you can find there active in the state you suggest, please, please do share that documentation, along with the information on your DNS servers and IP country location.


[ACTIVE.txt](https://github.com/tiuxo/hosts/files/7618420/ACTIVE.txt)
[INACTIVE.txt](https://github.com/tiuxo/hosts/files/7618421/INACTIVE.txt)

File: porn (hunk 2 above)
Comment (Rexadev) [reply porn:364]: for blocklist it doesn't matter. and this way its future proof.

**Comment by Rexadev** [comment]
> Just those I've marked from random picks, are turn up as dead domains. so how have you been able to determine they should be blacklisting (blocking) anything just by flipping the the usage of `www.`??? I'm really interested as your suggestion are the opuses of any test I can provide.

for blocklist removing www. doesn't matter and this way its future proof for example if sites starts using www1.

**Comment by Rexadev** [comment]
@spirillen Why you didn't add any of my suggestions to the blocklist?

**Comment by spirillen** [comment]
> for blocklist removing `www.` doesn't matter and this way its future proof for example if sites starts using www1.

That is not true for host/pihole, please read [rfc:952](http://tools.ietf.org/html/rfc952) as this is a host file.

Where you argumentation can be right is solely to software that do operate and understand the wildcard like [DNS RPZ.](https://mypdns.org/mypdns/support/-/wikis/RPZ-record-types) Here you will be able to read-in a source and append the `*.` _BUT ONLY IF_ and _WHEN_ the hole list is to be trusted as wildcard capable like mine `wildcard.list` this list is not such a list, it clearly says `hosts` where only exact matches gives points.

Rules for building PI-hole lists: https://github.com/pi-hole/pi-hole/issues/4431

> @spirillen Why you didn't add any of my suggestions to the blacklist?

1. Because this is not my project, it is @brianclemens and others project.
2. These giant lists like https://github.com/tiuxo/hosts/pull/33 takes me up to 6/7 hours to import, by first visiting the site, verify the information etc. etc.: https://github.com/blocklistproject/Lists/issues/565#issuecomment-979670619
3. I currently actually have imported 45 of your domains. https://mypdns.org/my-privacy-dns/porn-records/-/issues?scope=all&state=closed&search=github.com%2Ftiuxo%2Fhosts%2Fpull%2F33 + 4 from another pr https://mypdns.org/my-privacy-dns/porn-records/-/issues?scope=all&state=closed&search=https%3A%2F%2Fgithub.com%2Ftiuxo%2Fhosts%2Fpull%2F30
4. My question is then to ask you why you do not report them to [my project](https://mypdns.org/my-privacy-dns/porn-records) if you like me to blacklists them?. I have no power here,

**Pull request closed without merging**